src/user_api_spec.yaml

```diff
@@ -0,0 +1,186 @@
+swagger: '2.0'
+info:
+  description: >-
+    This is API for Direction Guesser using Swagger.  You can find out more about
+    Swagger at [http://swagger.io](http://swagger.io) or on [irc.freenode.net,
+    #swagger](http://swagger.io/irc/).
+  version: 1.0.1
+  title: Direction Guesser
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+host: direction.guesser.io
+basePath: /v2
+tags:
+  - name: users
+    description: User related calls.
+schemes:
+  - https
+paths:
+  /users/:
+    get:
+      tags:
+        - users
+      summary: Returns a list of all usernames.
+      operationId: getUsers
+      responses:
+        '200':
+          description: List of usernames.
+          schema:
+            type: array
+            items:
+              type: string
```

Comment:
I don't think this should not be a public API call, why would we want clients to be able to request a list of all users?

Reply:
Just rounds off the API, it's an operation you'd expect to have. Maybe it can be used for profile search on the client side, if we want something like that.

Reply:
If you want search, it should be done on the backend. This has potential to be a super expensive call / dos vector depending on scale, if we have a million users its sending back MB of data per request, which could easily overload network bandwidth if even a few users spammed it.

Reply:
> if we have a million users its sending back MB of data per request
When we get at that point we do what everyone else does and implement pagination in the API.

Reply:
Not against axeing it though, I didn't have any particular use for it in mind, other than not having it just feels weird from purely an API perspective.

If nobody else thinks of a concrete use, lets kill it.

Reply:
Meeting concluded to Axe this call.

Reply:
WILL BE REMOVED.